src/corvus_python/pyspark/utilities/session_utils/local_spark_session.py

```diff
@@ -0,0 +1,107 @@
+"""Copyright (c) Endjin Limited. All rights reserved."""
+
+from dataclasses import dataclass
+from pyspark.sql import SparkSession
+from delta import configure_spark_with_delta_pip
+import os
+from ...storage import StorageConfiguration, LocalFileSystemStorageConfiguration
+
+
+CWD = os.path.join(os.getcwd())
+
+
+@dataclass
+class LocalSparkSessionConfig():
+    """Class to represent configuration of a Local Spark setup.
+
+    Args:
+        workload_name (str): The name of the workload. This will be used as the name of the Spark application.
+        file_system_configuration (StorageConfiguration, optional): The storage configuration to use for the Spark
```

Comment:
Should this argument be called `storage_configuration`?

Reply:
Done